modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -0,0 +1,38 @@
+# == Define: govuk::apps::email_alert_api::sidekiq_queue_checks
+#
+# Creates an Icinga check which checks Graphite for activity
+# on Sidekiq queues
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The name of the sidekiq queue to check.
+#
+# [*size_warning*]
+#   The number of messages on a queue to trigger a warning.
+#
+# [*size_critical*]
+#   The number of messages on a queue to trigger a critical alert.
+#
+define govuk::apps::email_alert_api::sidekiq_queue_check(
+  $ensure = present,
+  $size_warning = undef,
+  $size_critical = undef,
```

Comment:
These are probably better as mandatory parameters, as they're required by the `icinga::check::graphite` manifest I think.